.github/workflows/grass-manual.yml

```diff
@@ -0,0 +1,7 @@
+name: GRASS GIS addon manual
+
+on: [push]
```

Comment:
I mentioned this in the github-workflow PR as well: Would it make sense to only run for `main` branch?